src/psyclone/transformations.py

```diff
@@ -361,17 +364,46 @@ class OMPTaskloopTrans(ParallelLoopTrans):
     >>> schedule.view()
 
     '''
-    def __init__(self, grainsize=None, num_tasks=None):
+    def __init__(self, grainsize=None, num_tasks=None, nogroup=False):
         self._grainsize = None
         self._num_tasks = None
         self.omp_grainsize = grainsize
         self.omp_num_tasks = num_tasks
-
+        self.omp_nogroup = nogroup
         super(OMPTaskloopTrans, self).__init__()
 
     def __str__(self):
         return "Adds an 'OpenMP TASKLOOP' directive to a loop"
 
+    @property
+    def omp_nogroup(self):
+        '''
+        Returns whether the nogroup clause should be specified for
+        this transformation. By default the nogroup clause is applied.
+
+        :returns: Whether the nogroup clause should be specified by
+                  this transformation.
+        :rtype: bool
+        '''
+        return self._nogroup
+
+    @omp_nogroup.setter
+    def omp_nogroup(self, nogroup):
+        '''
+        Sets whether the nogroup clause should be specified for this
+        transformation.
+
+        :param nogroup: bool value to set whether the nogroup clause
+                        should be used for this transformation.
+        :type value: bool
```

Comment:
Inline type spec. please (`:param bool nogroup: value to set whether...`)

Reply:
Done